src/viam/sdk/robot/client.cpp

```diff
@@ -222,7 +216,7 @@ void RobotClient::refresh_every() {
     }
 };
 
-RobotClient::RobotClient(std::shared_ptr<ViamChannel> channel)
+RobotClient::RobotClient(const std::shared_ptr<ViamChannel>& channel)
```

Comment:
Could also leave it a value and `std::move` it on 220 probably.

Reply:
Ahh good catch, thanks! Fixed.